printer_copy_number/views/printing_report_xml_action.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data noupdate="0">
```

Comment:
Replace `openerp` and `data` nodes by a single `odoo` node.
Specifying `noupdate="0"` is not needed (but not a problem).